### ISSUE TITLE
Suggestion: easier handling for manual input

### ISSUE THREAD

**Issue by XanGit** [ko-vasilev/Hearthstone-Collection-Tracker#53]
First, let me thank you very much for this addon, which works very well with HDT on my (german) win7/64 PC. Really helpful when importing and building decks in HDT.

As a F2P only player I'm far away from having a complete collection, therefor I would like to make some suggestions for those, who a still working on building their collection:

1. I would like to see an option, which sums my normal and golden cards and show only those as "missing", where I really have only one card. I know I can switch between normal and golden, however I really don't bother with "all golden" right now. May be you can make the "desired amount" counter to be something like 3-4 = also 1 or two golden and then only counts the sum. For the people which are on the "full collection mission" they can always input a 4 as desired amount for every card.

2. I would like to see both the counter for normal and golden cards in my collection on the same pane with the "sum".

3. If there was this kind of combined normal and golden number there could also be a "Show cards with more than desired amount"option. This would be handy if I want to disenchant some cards I have more than 2 of combined to make some cards I'm missing and only have 0 or 1.

4. For the monthly bonus cards or if I opened several packs (say after multiple arena runs) I sometimes don't know if the cards belong to "Classic", GvG or TgT. I usually open multiple packs (from different sets), then go to the My Collection tab in Hearthstone and look for "new" cards. I would like to be able to just type in some characters of the card name, but I have to choose which "set collection" to manage. It looks like your addin knows, which card belongs to which set and it would make things easier to only have one "Manage collection" button

5. Would it be possible to use the "Primary/Alternative Card Language" setting from HDT? I set the primary language to german, and the alternative language to english, and in HDT I'm able to search both for the german and the english card names. In Collection tracker the primary language is used, but not the alternative (as far as I have tested). If I switch the primary language in HDT to english, Collection manager uses (and searches for) the english card names, switch back to german and Collection manager use the german names.

Thanks again for this helpful addin, and thank you very much fpr reading and considering theses suggestions.

**Comment by ko-vasilev** [comment]
Hello and thank you. I want to notice that 'desired cards' feature does not care it the card is golden or non-golden so you will use this feature exactly if you're not aiming for full golden collection.
Yeah I think it will be a good thing to have something like "show excess cards".
Also being able to add a card without knowing its set also a nice feature.
I did not bother about languages because you're the first one who said that that is an issue. I will try to fix that.

**Comment by XanGit** [comment]
May be I'm doing something wrong or I don't understand the "desired amount" feature correct.

I have added a screenshot of an example: I have one copy of each a normal and a golden "Fel Guard" (Teufelswache in german) and set the desired amount to 2. If I select "Show only missing" I see this card both in normal and golden, but I would expect that it isn't shown.

As you can see in the screenshot I also have also 2 normal/1 golden of "Verdammniswache" or "Wache von Silbermond". If desired amount is set to 2 both are not shown for normal, but for golden cards although the added amount is 3.

![collectionmanager-01](https://cloud.githubusercontent.com/assets/15522731/11917313/91d8a9ce-a702-11e5-839e-cf5858cd2346.png)

Therefor I thought the "desired amount" setting (the dropdown) was separated for both the normal and the golden cards and not the sum of all cards. 

If I understand you correctly you say the desired amount number *should* work as a overall sum of the card. If this is the intended feature function then it looks like a bug. I'm using version 0.13.7 of HDT with 0.2.4 of collection manager.

**Comment by ko-vasilev** [comment]
Right, I just did not tie "show only missing" filter with this feature.
